ibc.ts/src/common/chain.ts

```diff
@@ -54,17 +54,34 @@ export class Chain {
     }
```

Comment:
If we send transactions without waiting for inclusion, Foundy can
create a block that has more than one transaction.
->
If we send transactions without waiting for inclusion, Foundry can
create a block that has more than one transaction.